backend/apps/cache/services/cache_manager.py

```
@@ -346,17 +346,27 @@ def invalidate_cache_on_save(sender, instance, created, **kwargs):
     """
     Invalidate cache when a model is saved.
     """
+    if sender._meta.app_label in ["migrations", "contenttypes", "sessions", "admin", "sites", "auth"]:
+        return
     manager = CacheManager()
-    manager.invalidate_dependencies(instance)
+    try:
+        manager.invalidate_dependencies(instance)
+    except Exception:
+        pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)